### DISCUSSION TITLE
Prisma Multi-Model problems

### DISCUSSION THREAD

**Discussion by geenva** [prisma/prisma#10472] [Q&A] [answered]
Hi, I have two models which are not related to one another: 
```prisma
model Newsletter {
  email String @unique
}

model User {
  id              Int           @id @unique @default(autoincrement())
  config          Configuration @relation(fields: [configurationId], references: [id])
  records         Record[]
  information     Information   @relation(fields: [informationId], references: [dbid])
  informationId   String
  affirmations    String?
  configurationId String
  recaps          Recap[]
  lastIndex       Int?
}
```

I create a new Newsletter record by running:
```ts
return await server.db.newsletter.create({
          data: {
            email: args.email,
          },
        });
```

It returns:
```
PrismaClientKnownRequestError: 
Invalid `prisma.newsletter.findFirst()` invocation:


  Failed to validate the query: `Field does not exist on enclosing type.` at `Query.findFirstNewsletter`
    at cb (/Users/*/*/*/*/node_modules/@prisma/client/runtime/index.js:38675:17)
    at Object.addEmailToNewsletter (/Users/*/*/*/*/src/resolvers/addEmailtoNewsletter.ts:7:16) {
  code: 'P2009',
  clientVersion: '3.5.0',
  meta: {
    query_validation_error: 'Field does not exist on enclosing type.',
    query_position: 'Query.findFirstNewsletter'
  }
}
```

Any idea how to resolve this?

**Comment by austincrim** [comment]
Hey @geenva 👋,

Just to confirm, you get the above error when running `db.newsletter.create()`? That seems strange because the error mentions `prisma.newsletter.findFirst()`.

**Reply by geenva** [reply]
Yes. However, I fixed it by downgrading to 3.0.2.

**Reply by janpio** [reply, marked as answer]
This is a known bug being tracked in https://github.com/prisma/prisma/issues/9553
Unfortunately we have no idea what is happening, so following the instructions in https://github.com/prisma/prisma/issues/9553#issuecomment-979383561 would be very welcome.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
